CTFd/forms/__init__.py

```diff
@@ -42,6 +43,7 @@ class _FormsWrapper:
 Forms.submissions = submissions
 Forms.users = users
 Forms.challenges = challenges
+Forms.lang = lang
```

Comment:
Should probably be called Forms.language. I am unsure if this definitely needs a new form.

Reply:
> I am unsure if this definitely needs a new form.

`forms.self.SettingsForm` cannot be used as it was designed to render data based on the logged in user.
`forms.users.UserEditForm/UserCreateForm` cannot be used as they need to refer to an existing user.

IMHO, we should keep the separate form as a cleaner solution.